articles/build-rules.md

```diff
@@ -0,0 +1,89 @@
+---
+authorName: Vassilis Rizopoulos
+authorGithubUsername: damphyr
+---
+# TBD
```

Comment:
tough one, brainstorming ideas:

- 4 rules to build heaven
- Your build system is probably messed up
- Why MSBuild, Gradle and Angular CLI are not build systems